### ISSUE TITLE
Legitimate site blocked: consortiumnews.com

### ISSUE THREAD

**Issue by Galeforce1** [phishfort/phishfort-lists#792]
Site is perfectly safe - it is a legitimate news site.

**Comment by shrestha-milann** [comment]
Hello,

The reported domain is not blacklisted by Phishfort. Could you please let
us know where you are seeing this?

Warm Regards,

Milan Shrestha

Anti-Phishing Operations

***@***.***

phishfort.com <https://www.phishfort.com/>

<https://www.linkedin.com/company/phishfort/>   <https://t.me/phishfort>
<https://twitter.com/PhishFort>

**Comment by Galeforce1** [comment]
Yes, I had a warning come up that told me there was a risk of infection if I proceeded, and it had 2 buttons with the options to proceed or not. I proceeded as I know it is quite safe and a legitimate news site.

There was a link to your github page, so I followed that and posted my issue..

.
I'm unable to reproduce it and send you a screenshot.
On 21/07/2023 11:24:45 AM, Milan Shrestha ***@***.***> wrote:
Hello,

The reported domain is not blacklisted by Phishfort. Could you please let
us know where you are seeing this?

Warm Regards,

Milan Shrestha

Anti-Phishing Operations

***@***.***

phishfort.com <https://www.phishfort.com/>

<https://www.linkedin.com/company/phishfort/> <https://t.me/phishfort>
<https://twitter.com/PhishFort>

**Comment by shrestha-milann** [comment]
I hope the issue has been resolved.
If you see the warning message again, please let me know.

Warm Regards,

Milan Shrestha

Anti-Phishing Operations

***@***.***

phishfort.com <https://www.phishfort.com/>

<https://www.linkedin.com/company/phishfort/>   <https://t.me/phishfort>
<https://twitter.com/PhishFort>

**Comment by Galeforce1** [comment]
Hi Milan,
The issue hasn't been resolved, it has happened again. I attach a screenshot
Regards,
William
[image: image.png]

**Comment by shrestha-milann** [comment]
Hello,

The mentioned domain wasn't blacklisted by Phishfort.
Please confirm by following the lookup service.
https://lookup.phishfort.com/api/lookup?domain=consortiumnews.com

Warm Regards,

Milan Shrestha

Anti-Phishing Operations

***@***.***

phishfort.com <https://www.phishfort.com/>

<https://www.linkedin.com/company/phishfort/>   <https://t.me/phishfort>
<https://twitter.com/PhishFort>
